content/en/blog/2022/end-user-discussion-group-apac/index.md

```diff
@@ -53,16 +53,16 @@ Slack `#opentelemetry` channel.
 
 Upcoming sessions:
 
-- **EMEA**: every third Tuesday of the month at 11AM CET (GMT +1), join
-  [here](https://us06web.zoom.us/j/85691064809?pwd=c0VCejh)
+- **[EMEA](https://us06web.zoom.us/j/85691064809?pwd=c0VCejh)**: every third
+  Tuesday of the month at 11AM CET (GMT +1)
```

Comment:
This change is more than spelling, it changed the way how words got structured.

Reply:
That's true. The rules here just take advantage of the fact that the tool is reading all the content and recognizing common problematic behaviors.